scripts/dev.js

```diff
@@ -16,11 +18,10 @@ let electronProcess = null
  * Start electron process and inspect port 5858 with 9222 as debug port.
  */
 function startElectron() {
-  /**
-   * @type {any}
-   */
-  const electronCommand = electron
-  const process = spawn(electronCommand, ['--inspect=5858', '--remote-debugging-port=9222', join(__dirname, '../dist/electron/index.dev.js')])
+  electronProcess = spawn(
+    /** @type {string} */electron,
```

Comment:
The vscode seems still report error here to me. Maybe it's better to keep old one?

Reply:
Unfortunately, I do not have the ability to test this in the Sun code (I am one of the few who do not use this editor). I think it will be safer if you fix it yourself.